# Generated by Django 3.2.12 on 2022-03-15 05:48

import django.core.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('menu', '0005_auto_20220315_0547'),
    ]

    operations = [
        migrations.AlterField(
            model_name='category',
            name='name',
            field=models.CharField(error_messages={'unique': 'That category already exists'}, max_length=255, unique=True, validators=[django.core.validators.MinLengthValidator(3)]),
        ),
        migrations.AlterField(
            model_name='ingredient',
            name='name',
            field=models.CharField(error_messages={'unique': 'That ingredient already exists'}, max_length=255, unique=True, validators=[django.core.validators.MinLengthValidator(3)]),
        ),
    ]
